build-aux/snap/snapcraft.yaml

```diff
@@ -29,173 +45,723 @@ license: GPL-3.0
 # See the comments from jdstrand in
 # https://forum.snapcraft.io/t/5547/10
 parts:
-  snapd-deb:
+  build-base:
+    plugin: nil
+    source: keyrings
+    build-packages:
+      - to i386:
+        - debootstrap
+        - fakeroot
+      - else:
+        - wget
+      - gawk
+    build-environment:
+      - RELEASE: "22.04.2"
+      - BASE: ubuntu-base-${RELEASE}-base-${CRAFT_TARGET_ARCH}.tar.gz
+      - DIR_URL: https://cdimage.ubuntu.com/ubuntu-base/releases/${RELEASE}/release
```

Comment:
i still not sure why this is created, but using ubuntu-base base tarballs here is probably not the best, given clearly you are trying to create a buildd image.

I would expect you to use our buildd images from here https://cloud-images.ubuntu.com/buildd/releases/jammy/release/ which is what launchpad & snapcraft use, and note they are provided for our partial architectures too such as i386.